administration/config/webpack.config.ts

```diff
@@ -230,7 +227,7 @@ function createWebpackConfig(webpackEnv: 'development' | 'production'): webpack.
               'scheduler/tracing': 'scheduler/tracing-profiling',
             }
           : {}),
-        ...(modules.webpackAliases && modules.webpackAliases.src ? modules.webpackAliases : {}),
+        ...(modules.webpackAliases.src ? modules.webpackAliases : {}),
```

Comment:
Is it ensured that webpackAliases cannot be null?

So, if modules.webpackAliases is undefined or null, the old expression will return {}, whereas the new will throw an error because it tries to access .src on an undefined or null value.

Reply:
Yes, I fixed the fixed, is always guaranteed to be an object.